src/_premake_init.lua

```diff
@@ -1113,6 +1111,40 @@
 		linktimeoptimization("Default")
 	end)
 
+	api.deprecateValue("flags", "FatalWarnings", "Use `fatalwarnings` instead.",
```

Comment:
Should these perhaps say explicitly what settings to use? ``"Use `fatalwarnings { "Compile", "Link" }` instead."``